File: auto-docs/executables/r/overlaid_histogram.r
# Learn about API authentication here: plot.ly/r/getting-started
# Find your api_key here: plot.ly/settings/api

library(plotly)

x0 = rnorm(500)
x1 = rnorm(500)+1

py <- plotly(username='TestBot', key='r1neazxo9w')
trace1 <- list(
  x = x0, 
  opacity = 0.75, 
  type = "histogram"
)
trace2 <- list(
  x = x1, 
  opacity = 0.75, 
  type = "histogram"
)
data <- list(trace1, trace2)
layout <- list(barmode = "overlay")
response <- py$plotly(data, kwargs=list(layout=layout, filename="overlaid-histogram", fileopt="overwrite"))
url <- response$url

File: auto-docs/executables/julia/overlaid_histogram.jl
# Learn about API authentication here: {BASE_URL}/julia/getting-started
# Find your api_key here: {BASE_URL}/settings/api

using Plotly

x0 = randn(500)
x1 = randn(500)+1

Plotly.signin("TestBot", "r1neazxo9w")
trace1 = [
  "x" => x0, 
  "opacity" => 0.75, 
  "type" => "histogram"
]
trace2 = [
  "x" => x1, 
  "opacity" => 0.75, 
  "type" => "histogram"
]
data = [trace1, trace2]
layout = ["barmode" => "overlay"]
response = Plotly.plot(data, ["layout" => layout, "filename" => "overlaid-histogram", "fileopt" => "overwrite"])
plot_url = response["url"]
